WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -123,6 +123,16 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [self setupPingHub];
     [self setupShortcutCreator];
     [self setupBackgroundRefresh:application];
+    
+    NSArray *args = [NSProcessInfo processInfo].arguments;
+    
+    for (NSString *arg in args){
```

Comment:
It will be nice to create a separate method for this code, and document what we are doing with it. I'm assuming you are making the animations faster in order to have tests faster.